recipes/repository.rb

```diff
@@ -27,6 +27,16 @@
 
 agent_major_version = Chef::Datadog.agent_major_version(node)
 
+apt_gpg_keys = ['A2923DFF56EDA6E76E55E492D3A80E30382E94DE', 'D75CEA17048B9ACBF186794B32637D44F14F620E']
+rpm_gpg_keys = [['DATADOG_RPM_KEY_E09422B3.public', 'e09422b3', 'A4C0 B90D 7443 CF6E 4E8A  A341 F106 8E14 E094 22B3'], 
+                ['DATADOG_RPM_KEY_20200908.public', '20200908', 'C655 9B69 0CA8 82F0 23BD  F3F6 3F4D 1729 FD4B F915']]
```

Comment:
We use the second parameter for both the attribute name (`yumrepo_gpgkey_new_20200908`) and the rpm public key package query (`rpm -q gpg-pubkey-#{rpm_gpg_key[rpm_gpg_keys_pubkey]}`).

Either we do this:
```suggestion
                ['DATADOG_RPM_KEY_20200908.public', 'fd4bf915', 'C655 9B69 0CA8 82F0 23BD  F3F6 3F4D 1729 FD4B F915']]
```

and`yumrepo_gpgkey_new_20200908` attribute needs to be changed into `yumrepo_gpgkey_new_fd4bf915` (and some tests will have to be changed to match).

Or we have two separate parameters, one for the date, the other for the short fingerprint, like this:
```suggestion
                ['DATADOG_RPM_KEY_20200908.public', '20200908', 'fd4bf915', 'C655 9B69 0CA8 82F0 23BD  F3F6 3F4D 1729 FD4B F915']]
```

in which case we'll have to cheat for the `e09422b3` key by putting `e09422b3` as the date (if we want the below logic to work).


I'd recommend the first solution, which feels less hacky.